{"uuid":"53b7230e-0348-4ca9-a415-91b326014dce","name":"Scenario Outline: The user is presented with error message for code with invalid syntax in tryEditor for Arrays in Python page","children":["e5b1e6f3-780b-48a3-b13c-2d5e0ed9b1fb"],"befores":[],"afters":[{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[{"name":"My screenshot","source":"3953c3ef-5824-4171-9cca-1eeb4e376143-attachment","type":"image/png"},{"name":"Myscreenshot","source":"7743f94a-9c58-4913-afcc-8fed56620560-attachment"}],"parameters":[],"start":1696884704281,"stop":1696884704661},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696884704664,"stop":1696884704665},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696884704667,"stop":1696884704667},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696884704669,"stop":1696884704669}],"start":1696884704263,"stop":1696884704670}